### DISCUSSION TITLE
❓ Questions / Help / Support претренированная модель STT для русского языка - доступность, лицензия, etc

### DISCUSSION THREAD

**Discussion by MrCatEsq** [snakers4/silero-models#123] [Q&A]
В репозитории нет, в FAQ единственное [упоминание](https://github.com/snakers4/silero-models/wiki/FAQ#where-can-i-find-a-russian-model) ведет на сайт, где нет подробностей. Есть ли/предполагается ли публикация в свободном доступе STT модели русского языка для offline/standalone/selfhosted использования? Где можно ознакомиться с ценами, если она предоставляется только платно (если они доступны публично).

**Comment by snakers4** [comment]
Привет, пока таких планов нет.

**Reply by MrCatEsq** [reply]
Есть какая-то платная лицензия, доступная для частных лиц или небольших организаций? Или хотя бы +- пошаговый гайд для самостоятельного обучения модели на основе open_stt и вашего собственного тулкита или deepspeech2, например?
